src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -10,17 +10,14 @@ namespace System.Security.Cryptography.Cose
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct CoseHeaderLabel : IEquatable<CoseHeaderLabel>
     {
-        internal string LabelName => LabelAsString ?? LabelAsInt32.ToString();
+        internal string LabelName => $"\"{LabelAsString}\"" ?? LabelAsInt32.ToString();
```

Comment:
I feel like I saw a comment from @vcsjones about this, but it's not showing up in the diff view...

The null coalesce here is impossible, since $"" will always produce a non-null string.

Reply:
Yup, fixed.